builtin/credential/aws/path_login.go

```diff
@@ -19,6 +19,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hashicorp/go-secure-stdlib/parseutil"
+
```

Comment:
Would it be good to remove this extra line?

Reply:
Good call, it was added by my IDE but it definitely doesn't look correct there, looking at the full file. I'll fix these!